tools/flaky-test-retryer/github_commenter.go

```diff
@@ -33,6 +33,8 @@ type GithubClient struct {
 
 func NewGithubClient(githubAccount string) (*GithubClient, error) {
```

Comment:
You'll need a comment on top of this or lint is gonna complain

Reply:
done